lib/manager/gradle-wrapper/artifacts.ts

```diff
@@ -37,7 +37,9 @@ export async function updateArtifacts({
       gradlew,
       projectDir,
       await fs.stat(gradlewPath).catch(() => null),
-      `wrapper --gradle-version ${config.toVersion}`
+      `wrapper --gradle-version ${config.toVersion.substring(
+        config.toVersion.lastIndexOf('.0')
```

Comment:
This doesn't quite work. If `toVersion` is `6.3.0`, this returns `.0`. If we use `config.toVersion.substring(0, config.toVersion.lastIndexOf('.0'))` it correctly returns `6.3`, but if `toVersion` is `6.0.1` then it returns `6`.

At a higher level, what you're trying to do, is find versions with a patch version of `0`, and only return `vMajor.vMinor` in those cases. I would look at using [semver](https://github.com/renovatebot/renovate/blob/master/lib/versioning/semver/index.ts) to handle the abstraction instead of directly doing string manipulation.